test/cloud.jl

```diff
@@ -0,0 +1,13 @@
+using Base.Test
+using Gurobi
+using MathProgBase.SolverInterface
+
+env = Gurobi.Env("csdemo.gurobi.com")
+Gurobi.free_env(env)
```

Comment:
Yes, @cmaes suggested we solve small problems on the cloud so that we can test Gurobi.jl on Travis.